maven-modularity-plugin/pom.xml

```diff
@@ -82,32 +82,32 @@
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-plugin-api</artifactId>
-            <version>2.0.4</version>
+            <version>3.5.3</version>
         </dependency>
-        <dependency>
+        <!--<dependency>
```

Comment:
1. Remove lines instead of commenting
2. Update copyright year since you are modifying this file.